styles/style.css

```diff
@@ -9,25 +9,28 @@ body {
     height: 100%;
     display: grid;
     grid-template-columns: 25% 50% 25%;
-    grid-template-rows: 25% 50% 25%;
+    grid-template-rows: 100px 800px 100px;
     padding-left: 10px;
     text-align: center;
     padding-top: 50px;
     background-color: black;
     color: white;
+    font-size: 1.5rem;
 
```

Comment:
more formatting things to look out for -- removing empty lines to clean up the code a bit (not removing points for these, just pointing them out for the future)